ddl/resource_group_test.go

```diff
@@ -176,3 +176,21 @@ func testResourceGroupNameFromIS(t *testing.T, ctx sessionctx.Context, name stri
 	g, _ := dom.InfoSchema().ResourceGroupByName(model.NewCIStr(name))
```

Comment:
Please move it into ```ddl/resourcegrouptest```If possible.

the test cases in the ddl are too large to run as soon as possible.

Reply:
ref #41377